Anführungsstriche und Textänderungen

AurelioSilver opened the pull request.

AurelioSilver committed:
Item fixes by  Cynthel Natal

AurelioSilver committed:
Anführungstriche Durgans Battery Teil 1

AurelioSilver committed:
Anführungstriche Durgans Battery Teil 2

AurelioSilver committed:
Code page fix

AurelioSilver committed:
Anführungszeichen: px1_03_wilderness

AurelioSilver committed:
Anführungszeichen: px1_04_mercenary_camp

AurelioSilver committed:
Anführungsstriche: conversatiuons/px1_companions

AurelioSilver committed:
Anführungstriche: conversations\px1_companions\poi

AurelioSilver committed:
Anführungsstriche: px1_00_stalwart_village

AurelioSilver committed:
Anführungsstriche: quests

AurelioSilver committed:
Anführungsstriche: /game/ Teil1

AurelioSilver committed:
Anführungsstriche: /game/ Teil 2

AurelioSilver committed:
Korrektur

AurelioSilver committed:
Versperrende Tür des Todes -> Gnadenfrist

AurelioSilver commented:
So, sollte mit den Änderungen nun durch sein, außer es fallen noch Korrekturen an.
Gedankenstriche habe ich nicht bearbeitet. Wäre eher was für nkemnitz.

Wer mag querlesen und mergen? :)

AurelioSilver committed:
Patch 2.0.2 Changes to conversation

AurelioSilver committed:
Patch 2.0.2 changes to afflictions

AurelioSilver committed:
Patch 2.0.2 changes to GUI

Xaratas commented:
Ab conversation fehlen noch die folgenden 6, Rest ist mit Anmerkungen versehen oder io.

AurelioSilver committed:
Anführungszeichen: Korrekturen

AurelioSilver committed:
Korrektur zurückgezogen.

Xaratas commented:
Alle durch.

AurelioSilver committed:
Fehlende Anführungsstriche ergänzt

AurelioSilver committed:
Patch 2.0.2 Beta 2 Changes
Occlusion Opacity
Adjust the opacity of characters when they are occluded by the environment.
